webbpsf/opds.py

```diff
@@ -1716,16 +1735,26 @@ def move_sur(self, sur_file, group=None, verbose=False):
                     rot_unit = update.units['X_TILT']
                     trans_unit = update.units['X_TRANS']
 
-                    self.move_seg_local(update.segment[0:2],
-                                        xtilt=update.moves['X_TILT'],
-                                        ytilt=update.moves['Y_TILT'],
-                                        xtrans=update.moves['X_TRANS'],
-                                        ytrans=update.moves['Y_TRANS'],
-                                        piston=update.moves['PISTON'],
-                                        absolute=update.absolute,
-                                        rot_unit=rot_unit,
-                                        trans_unit=trans_unit,
-                                        delay_update=True)
+                    if update.segment == 'SM':
+                        self.move_sm_local(xtilt=update.moves['X_TILT'],
+                                           ytilt=update.moves['Y_TILT'],
+                                           xtrans=update.moves['X_TRANS'],
+                                           ytrans=update.moves['Y_TRANS'],
+                                           piston=update.moves['PISTON'],
+                                           rot_unit=rot_unit,
```

Comment:
It looks like the `update.absolute` parameter is not passed through to the SM move command. I'm guessing this is an unintentional omission? I can't remember the details but I assume that absolute moves are allowed for the SM just as for any primary segment.